### DISCUSSION TITLE
joltc api parity

### DISCUSSION THREAD

**Discussion by oh-facts** [jrouwe/JoltPhysics#832] [Q&A]
Sorry that this might not be the right place. But if someone has used the joltc api ([this one](https://github.com/amerkoleci/JoltPhysicsSharp/tree/main/src/joltcl)), could they let me know how up to date it is with jolt physics currently? What features aren't present, etc. If its too far off, would it be unreasonably long to make a nice C api? If not, could I get some direction on how I could start? 
Thank you!

**Comment by Srekel** [comment]
You can find a C API for Jolt here (and Zig bindings on top of them).

https://github.com/michal-z/zig-gamedev
https://github.com/michal-z/zig-gamedev/tree/main/libs/zphysics
https://github.com/michal-z/zig-gamedev/tree/main/libs/zphysics/libs/JoltC

They aren't complete but get PRs every now and then.
